ServerlessLibraryAPI/wwwroot/items.json

```diff
@@ -526,7 +526,7 @@
     "language": "javascript",
     "type": "functionapp",
     "authortype": "Community",
-    "runtimeversion": "na"
+    "runtimeversion": "v2"
```

Comment:
as per the deployment template, this should be v1

Reply:
I went by the v2.0 mentioned here https://github.com/0GiS0/azure-functions-loves-twilio/blob/master/host.json